packages/faustwp-cli/src/telemetry/sendTelemetryData.ts

```diff
@@ -1,39 +1,23 @@
 import 'isomorphic-fetch';
 
+import { getWpUrl } from '../utils/getWpUrl.js';
+import { getWpSecret } from '../utils/getWpSecret.js';
 import { TelemetryData } from './marshallTelemetryData.js';
-
-const GA_TRACKING_ENDPOINT = 'https://www.google-analytics.com/mp/collect';
-const GA_TRACKING_ID = 'G-KPVSTHK1G4';
-const GA_API_SECRET = '-SLuZb8JTbWkWcT5BD032w';
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+const WP_TELEMETRY_ENDPOINT = `${getWpUrl()!}/wp-json/faustwp/v1/process_telemetry`;
```

Comment:
It looks like the requests that are being sent are failing because `getWPUrl()` is returning nothing here. (You can confirm this by commenting out the `console.log` in the `catch` block [here](https://github.com/wpengine/faustjs/pull/1687/files#diff-8c3b33235dbe1bbcfadb06a8c95846376dbe4999ceefe0276219ba239740119fR80).

This is because the `dotenv` config is being instantiated from within the [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE#) in the index file, and `WP_TELEMETRY_ENDPOINT` is defined outside of it. You can move `WP_TELEMETRY_ENDPOINT` within the `sendTelemetryData` function and it should work as expected.